direct-hs/src/Web/Direct/Client/Channel/Types.hs

```diff
@@ -85,25 +82,9 @@ send chan msg = createMessage (channelRPCClient chan) msg $ channelTalkId chan
 
 ----------------------------------------------------------------
 
--- | Type of channel.
-data ChannelType = Pair               !User
-                 | PinPoint !TalkRoom !User
-                 | Group    !TalkRoom
-                 deriving (Eq, Show)
-
--- | Pair channel with the user.
---   A pair talk room is created if necessary.
---   The conversation is NOT seen by other users.
-pairChannel :: User -> ChannelType
-pairChannel user = Pair user
-
--- | One-to-one channel with the user in the talk room.
---   The conversation is seen by other users.
-pinPointChannel :: TalkRoom -> User -> ChannelType
-pinPointChannel room user = PinPoint room user
-
--- | Group channel in the talk room.
-groupChannel :: TalkRoom -> ChannelType
-groupChannel room = Group room
-
 type ChannelKey = (TalkId, Maybe UserId)
+
+data Partner =
```

Comment:
`Participants` seems to contain myself (bot).
I prefer `Partner` in it.

But there may be any other good terms.

Reply:
How about `Peer`?

Reply:
Okay, leave `Partner` so far (after oral discussion).
Rename if a better name is made up until it's published.

Reply:
After oral discussion again, we decided to use just `Maybe User` instead of creating its own type.
Then rename the argument name and describe it as it's the limitation to the participants of the channel.